bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/operation/SpecS2NSlitVisitor.java

```diff
@@ -195,9 +198,13 @@ private VisitableSampledSpectrum signal(final double throughput) {
 
         final VisitableSampledSpectrum signal = (VisitableSampledSpectrum) sourceFlux.clone();
         final int lastPixel = lastCcdPixel(signal.getLength());
+        Log.fine("Calculating signal with " + throughput + " throughput on detector pixels " + firstCcdPixel + " - " + lastPixel);
+
+        for (int i = 0; i < firstCcdPixel; ++i) { signal.setY(i, 0); }
```

Comment:
If you think that's cleaner, I'd be happy with that.  I wasn't sure which would be faster.